bentoml/_internal/server/instruments.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Maybe lets not include this in the `grpc` PR, since this has nothing to do with gRPC.

You can make a separate PR to `main` for types updates.

Reply:
got it